.travis.yml

```diff
@@ -11,3 +11,4 @@ before_install:
   - sudo apt-get update -qq
   - sudo apt-get install libzmq3
   - sudo apt-get install libzmq3-dev
+  - gem update bundler
```

Comment:
IMHO it should be something like https://github.com/mdsol/medistrano/blob/develop/.travis.yml#L114 , otherwise whenever a new version is out you'll be, whether you want it or not, upgraded. When 1.11.0 was released it broke jruby builds for instance. And they botched 1.11.1 trying to solve that, that's why we have 1.11.2 now.